src/styles/styles.css

```diff
@@ -0,0 +1,760 @@
+:root {
```

Comment:
[OPTIONAL] Your styling is looking great, excellent job! I would suggest splitting your CSS into several files; this way, you would avoid having one very long file, making it harder to handle.